.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number & title.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number & title.

From the result give me those PR assignees in a separate sentence, first combine the assignees from both bug and enhancement prs then sort them alphabetically.